velox/functions/sparksql/Bitwise.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <string>
+#include "velox/functions/Macros.h"
+namespace facebook::velox::functions::sparksql {
+
+void registerBitwiseFunctions(const std::string& prefix);
+
+template <typename T>
+struct BitwiseAndFunction {
```

Comment:
I guess these can be moved into Bitwise.cpp file.

Reply:
Yes, just refined the code layout. Thanks!